Kudu.Core/Deployment/FetchDeploymentManager.cs

```diff
@@ -255,6 +278,27 @@ public async Task PerformDeployment(DeploymentInfoBase deploymentInfo, IDisposab
             }
         }
 
+        private async Task SendDeployStatusUpdate(DeployStatusApiResult updateStatusObj, bool forceRestart)
+        {
+            int attemptCount = 0;
+            try
+            {
+                await OperationManager.AttemptAsync(async () =>
+                {
+                    attemptCount++;
+
+                    Console.WriteLine($" PostAsync - Trying to send {Constants.BuildFailed} deployment status to {Constants.UpdateDeployStatusPath}");
+                    await PostDeploymentHelper.PostAsync(string.Format(Constants.UpdateDeployStatusPath, forceRestart), _environment.RequestId, JsonConvert.SerializeObject(updateStatusObj));
```

Comment:
In Windows kudu the third parameter in PostAsync is hostName, defaults to null. So either add it as a null parameter here or leave it as default and use content as a named parameter.